first_last_position_lc34.py

```diff
@@ -0,0 +1,31 @@
+"""
+Author: Srinidhi
+Did the problem run on leetcode: No - I got TLE as I used brute force
+
+Time Complexity: 
+Space Complexity:
+
+Logic: Find the first and last position using linear scan and mark the first and last positions in the result array
+"""
+
```

Comment:
solve using the approach taught in class

Reply:
Yes Lohit, I will be updating this question soon.